core/src/main/java/it/pagopa/selfcare/mscore/core/ContractService.java

```diff
@@ -256,19 +256,18 @@ private NotificationToSend toNotificationToSend(Institution institution, Token t
             notification.setState(token.getStatus() == RelationshipState.DELETED ? "CLOSED" : token.getStatus().toString());
             // when update last update is updated date
             notification.setUpdatedAt(Optional.ofNullable(token.getUpdatedAt()).orElse(token.getCreatedAt()));
+            if (token.getStatus().equals(RelationshipState.DELETED)) {
```

Comment:
is this already unit tested?

Reply:
No, is tested with external integration test due to urgent release stable version for send data to data lake

Reply:
Test added on new release